.pre-commit-config.yaml

```
@@ -121,6 +121,15 @@ repos:
     - cheroot/
     pass_filenames: false
 
+- repo: local
+  hooks:
+  - id: stubtest
+    name: stubtest
+    entry: stubtest cheroot --mypy-config-file mypy.ini --allowlist stubtest_allowlist.txt
+    language: system
+    types: [python]
```

The preference for the sequences is to be multiline:
```suggestion
    types:
    - python
```